autogpt_platform/backend/backend/data/block.py

```diff
@@ -176,6 +176,11 @@ def __pydantic_init_subclass__(cls, **kwargs):
                 f"Field 'credentials' on {cls.__qualname__} "
                 f"must be of type {CredentialsMetaInput.__name__}"
             )
+        if credentials_field := cls.model_fields.get(CREDENTIALS_FIELD_NAME):
+            credentials_input_type = cast(
+                CredentialsMetaInput, credentials_field.annotation
+            )
+            credentials_input_type.validate_credentials_field_schema(cls)
```

Comment:
This causes the `events` input on the GitHub PR Trigger block not to show up:
1. `GithubTriggerBase.Input` gets initialized
    1. `validate_credentials_field_schema(input_schema)` calls `input_schema.jsonschema()`
    2. `input_schema.jsonschema()` sets `cls.cached_jsonschema`
2. `GithubPullRequestTriggerBlock.Input` gets initialized, and inherits `cached_jsonschema` from `GithubTriggerBase.Input`
3. When `GithubPullRequestTriggerBlock.input_schema.jsonschema()` gets called, it returns `GithubTriggerBase.Input.cached_jsonschema`, which lacks `events`

Reply:
This was so weird it took me an hour and a half to debug